lib/davinci_dtr_test_kit/dtr_light_ehr_suite.rb

```diff
@@ -71,5 +104,107 @@ class DTRLightEHRSuite < Inferno::TestSuite
             required_suite_options: DTROptions::SMART_2_REQUIREMENT,
             run_as_group: true
     end
+
+    group do
+      title 'FHIR API'
+
+      group from: :'us_core_v311-us_core_v311_fhir_api',
+            run_as_group: true
+    end
+
+    group do
+      title 'DTR Light EHR Profiles'
+
+      input :credentials,
+            title: 'OAuth Credentials',
+            type: :oauth_credentials,
+            optional: true
+
+      # All FHIR requests in this suite will use this FHIR client
+      fhir_client do
+        url :url
+        oauth_credentials :credentials
+      end
+
+      group do
+        title 'CRD Coverage'
```

Comment:
These should each be [run as group](https://github.com/inferno-framework/us-core-test-kit/blob/main/lib/us_core_test_kit/generated/v5.0.1/condition_encounter_diagnosis_group.rb#L80) since they assume scratch is populated by the first test.

Reply:
these other groups still need `run_as_group` like QuestionnaireResponse